utils/processmd.js

```diff
@@ -6,8 +6,10 @@ const files = readdirSync("./_posts/");
 
 async function main() {
   for (let file in files) {
+    const splitted = files[file].split(".");
+    const fileending = splitted[splitted.length - 1];
```

Comment:
using camel case here would make it more readable

Reply:
Good idea!